modules/images-describing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-describing-image-registry-settings-hcp_{context}"]
+= Describing image registry settings for {hcp-title}
+
+
+Description needed !!!!
```

Comment:
Just a reminder to add the description here.

Reply:
this module was removed, thanks